src/renderer/components/hotbar/hotbar-menu.tsx

```diff
@@ -73,17 +73,17 @@ class NonInjectedHotbarMenu extends React.Component<Dependencies & HotbarMenuPro
     this.props.hotbarStore.restackItems(from, to);
   }
 
-  removeItem(uid: string) {
+  removeItem = (uid: string) => {
```

Comment:
I don't like that these are no longer methods, if you want to pretend that JS allows referencing to methods like this I would recommend just calling `autobind(this)` in the constructor instead.